20180619/AGENDA.md

```diff
@@ -0,0 +1,7 @@
+# Sardana Follow-up Meeting - 2018/06/19
+
+To be held on 2018/06/19 at 14:00
+
+# Agenda
+1. Urgent issues/problems
+2. Preparation of the Sardana Documentation Camp (Alba-July18)
```

Comment:
For the point 2. What do you think if we quickly present the following things:
* Present structure of the current Sardana documentation (hirerarchy of chapters, important links between them, which chapters are empty, etc.).
* Present structure of the Sardana-DESY documentation (the same as previous point witht focus on the parts of the docs which are generic and could be moved to the Sardana documentation).
* Best practices when writing documentation (I found [Writing Great Documentation](http://www.writethedocs.org/guide/#writing-great-documentation) from www.writethedocs.org but couldn't read it yet), for sure there are more sources in the internet.
* Present a documentation strucuture of some other well documented project (maybe we could take a look on Tango which documentation was recently rewritten).

Or if you see other interesting things, just propose.

Reply:
I think the points you propose are fine.
With respect to the DESY documentation, I have had a look, and it is actually so much dependent
on our system, that I do not see anything that we could use as general ... not a whole section ...
 perhaps some paragraphs, but nothing really big.